src/api/app/views/webui/users/tokens/new.html.haml

```diff
@@ -7,24 +7,7 @@
 
     .row
       .col-12.col-md-10.col-lg-8
-        = form_with(model: @token, method: :post, local: false) do |f|
-          .visible-when-created.d-none
-            %p.font-weight-bold Your new OBS Personal Access Token
-
-            .form-row#token-id
-              .col-sm
-                .form-group.mb-0
-                  = f.label(:id, 'Id:')
-                  %span= @token.id
-
-            .form-row#created-token
-              .col-12.col-md-10.col-lg-9
-                %fieldset.form-group
-                  = f.label(:string_readonly, 'Secret:', class: 'col-form-label')
-                  .input-group
-                    = render CopyToClipboardInputComponent.new(form_builder: f)
-                %hr
-
+        = form_with(model: @token ||= Token.new, method: :post, local: true) do |f|
```

Comment:
[local: true](https://api.rubyonrails.org/v6.1.4/classes/ActionView/Helpers/FormHelper.html#method-i-form_with) is the default value, so it can be removed.

Reply:
The default value is `local: false`. It sets `remote: true`.

Reply:
Oh, I double-checked and you are right. Then it's all good :+1: